frontend/src/container/Account.jsx

```diff
@@ -48,7 +46,7 @@ import AgendaInfo from '../component/Dashboard/AgendaInfo.jsx'
 
 export class Account extends React.Component {
   constructor (props) {
-    super(props)
+    super()
```

Comment:
Why are you removing props here?

Reply:
See my other comment ;-)
```suggestion
    super(props)
```